### ISSUE TITLE
Range Selection

### ISSUE THREAD

**Issue by NoobWithBoobs** [vinnymac/PokeNurse#78]
Hello,
Range Selection would be a great feature

Examples
- 799 CP + 79 IV: Selects everything 799 CP & 79 IV and Under.
- 799 CP + 79 IV + Pokemon(s): Targets particular Pokemon(s) 
- 799 CP + 79 IV - Pokemon(s): Excludes particular Pokemon(s) 

What do you guys think?

**Comment by furier** [comment]
Yeah cool suggestion, for instance then you could transfer all pokemons bellow a CP cap and IV cap.
